Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -373,7 +373,7 @@ StyleAppearance RenderTheme::autoAppearanceForElement(RenderStyle& style, const
         auto& input = downcast<HTMLInputElement>(element.get());
 
         if (input.isTextButton() || input.isUploadButton())
-            return StyleAppearance::PushButton;
```

Comment:
Does this mean we can remove `StyleAppearance::PushButton` entirely?

Reply:
Yes, I'd prefer doing so in another bug, once we're sure we don't want to use it (in internal apps or such), so we can add an internal value later on if necessary.